tasks/ConfigurableSynchronizationTransmitter.cpp

```diff
@@ -0,0 +1,67 @@
+/* Generated from orogen/lib/orogen/templates/tasks/Task.cpp */
+
+#include "ConfigurableSynchronizationTransmitter.hpp"
+#include "Orocos.hpp"
+
+using namespace envire;
+
+ConfigurableSynchronizationTransmitter::ConfigurableSynchronizationTransmitter(std::string const& name)
+    : ConfigurableSynchronizationTransmitterBase(name), initial_map_sent(false),env(new envire::Environment()), envireEventDispatcher(NULL)
+{
+}
+
+ConfigurableSynchronizationTransmitter::ConfigurableSynchronizationTransmitter(std::string const& name, RTT::ExecutionEngine* engine)
+    : ConfigurableSynchronizationTransmitterBase(name, engine), initial_map_sent(false), env(new envire::Environment()), envireEventDispatcher(NULL)
```

Comment:
You might want to initialize the initial_map_sent flag in the configuration hook. Otherwise if the task is reused during task network transitions there wont be a map send out.